mimic/model/nova_objects.py

```diff
@@ -447,23 +507,29 @@ def create_success_report_failure_behavior(parameters):
     Create a behavior that reports failure, but actually succeeds, for server
     creation.
 
-    Takes two parameters:
+    Takes three parameters:
 
     ``"code"``, an integer describing the HTTP response code, and
     ``"message"``, a string describing a textual message.
+    ``"type"``, a string representing what type of error message it is
 
-    The response body will be a JSON object including ``code`` and ``message``
-    fields matching the parameters.
-    """
-    status_code = parameters.get("code", 500)
-    failure_message = parameters.get("message", "Server creation failed.")
+    If ``type`` is "string", the message is just returned as the string body.
+    Otherwise, the following JSON body will be synthesized (as per the
+    canonical Nova error format):
 
-    def create_then_fail(collection, http, json, absolutize_url):
-        Server.from_creation_request_json(
-            collection, json, lambda: randrange(255))
-        http.setResponseCode(status_code)
-        return dumps(invalid_resource(failure_message, status_code))
-    return create_then_fail
+    ```
```

Comment:
We should probably have a docstring linter.  I guess we already do in some sense, but there are very few things that are outright *errors* in ReST, just lots of things that format horribly. I don't think we have any online generation of docs I could point to to see how this formatted.